kas-poky-snapd.x86_64.yml

```diff
@@ -0,0 +1,5 @@
+header:
+  version: 8
+  includes:
+    - kas-poky-snapd.common.yml
```

Comment:
I used this approach so that we can vary more than just the machine type. For instance I want to run CI for specific kernel versions as well as for machines, and this mode is just less convoluted.